src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -242,23 +247,29 @@ class GcsServer {
   std::unique_ptr<GcsHealthCheckManager> gcs_healthcheck_manager_;
   /// The gcs redis failure detector.
   std::unique_ptr<GcsRedisFailureDetector> gcs_redis_failure_detector_;
+  /// The gcs placement group manager.
+  std::unique_ptr<GcsPlacementGroupManager> gcs_placement_group_manager_;
   /// The gcs actor manager.
   std::unique_ptr<GcsActorManager> gcs_actor_manager_;
   /// The gcs placement group scheduler.
   /// [gcs_placement_group_scheduler_] depends on [raylet_client_pool_].
   std::unique_ptr<GcsPlacementGroupScheduler> gcs_placement_group_scheduler_;
-  /// The gcs placement group manager.
-  /// [gcs_placement_group_manager_] depends on [gcs_placement_group_scheduler_].
```

Comment:
Key objects like gcs table storage is the dependency for multiple other data members, documenting every dependency edge makes code messy and hard to maintain.